samples/fabric/weaver/relay/README.md

```diff
@@ -1,12 +1,12 @@
-# Fabric to Fabric Interoperability view Weaver Relay 
+# Fabric to Fabric Interoperability view Weaver Relay
 
-[Weaver](https://labs.hyperledger.org/weaver-dlt-interoperability/docs/external/architecture-and-design/overview) 
+[Weaver](https://labs.hyperledger.org/weaver-dlt-interoperability/docs/external/architecture-and-design/overview)
```

Comment:
Why do we have this?

Reply:
I guess VSCode is automatically removing tailing spaces when saving markdown files :) - which seems to be a good thing :D

Reply:
aha. Does goland's have this linter? Can we include it in the CI?